notebooks/05_operationalize/dbapi_conda.yml

```diff
@@ -0,0 +1,17 @@
+

```

Comment:
Please see my comment above to see how we can simplify this

Reply:
100% agreed that simplifying would be great. This is what the implementation looks like when you follow all the steps that are already described in text.